src/longopt/grdproject_inc.h

```diff
@@ -25,32 +25,20 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
+	{ 0, 'C', "center",            "", "", "", "", GMT_TP_STANDARD },
+	{ '/', 'D', "increment|inc|spacing",
 	          "",                  "",
+	          "e,n",               "exact,number",
 		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'E', "dpi",               "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'F', "one2one|one2one_scale",
+	          "c,i,p,e,f,k,M,n,u", "cm,inch,point,meter,foot,km,smile,nmile,ussft",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
+	GMT_G_OUTGRID_KW,
+	{ 0, 'I', "inverse",           "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'M', "unit|projected_unit",
+	          "c,i,p",             "cm,inch,point",
```

Comment:
Similar to the above: could we do "cm,inch|in,point|pt"

Reply:
I am not sure if we want distance units to be spelled out.  That would mean hundred of places where we should allow km for k, point for p, etc, Think of map projections, e.g. Jm72point.  I think it is cleaner that people need to know what the unit abbreviations are instead of having to deal with smile and foot. I think the most commonly used units will be c, i, p, m, k.  Doubt anyone uses USGS survey foot. Mariners may use nautical miles.
However, I am happy to hear what more people think of this.  Seems like unnecessary and much work to mel  @joa-quim?, @seisman ?, @leouieda , @Esteban82 ?